### ISSUE TITLE
Does SC 3.3.9 Redundant Entry require the data available for the user to select available in the current step if the other steps are on the same page?

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2805]
Does SC 3.3.9 Redundant Entry require data available for the user to select be available with in the current step if the other steps are on the same page?  In addition, if the data is available but hidden behind an expand widget - is that a failure?

**Comment by alastc** [comment]
Hi Jon, we went through a long discussion of "steps", which was very hard to pin down as a concept. (Steps could be pages, or sections within a page etc.)

So we removed that, leaving the default of "page", as that is how SC are evaluated unless it says otherwise. 

So I would interpret "available for the user to select" as on the page, but could be behind a show/hide component.

**Comment by mraccess77** [comment]
Thanks Alastair.  The understanding document still refers to steps - so I thought it would be good to confirm that and the show/hide part as well.
